### DISCUSSION TITLE
How to access stored sessions of my users?

### DISCUSSION THREAD

**Discussion by rudrabhikadiya3** [vvo/iron-session#699] [Q&A] [answered]
I am using this in Nextjs page router. In order to access my user's sessions I have to see the stored session in server so I can access it.

In case, if the user is doing bad activity in my web app so I can delete his session so he will automatically logout and I will block him through the database then he retry to login then his status will be blocked.

**Comment by vvo** [comment, marked as answer]
Hey @rudrabhikadiya3, To achieve what you want ("Block user from logging in") then you need to:

- whenever your user is doing an API action on your website, check in your database if they have the rights to do so
- if they are blocked, refuse the request
- on login, make sure you check this field too
- in your admin interface, make it so you can block people
- done!
